ElrondNetwork/elrond-go
# Consensus const change

## Description of the reasoning behind the pull request
- during the operations of the backup nodes, there may be situations in which a backup node can not "see" the main node's messages because the rest of the network will actively drop the second consensus message. This will make the backup node to send more consensus messages until a message from the master machine is received.

## Proposed Changes
- make the consensus accept a maximum of 2 messages per public key, not just 1 as it was before

## Testing procedure
- standard system test
- a test without transactions, all nodes started with the following debug log level: `*:DEBUG,consensus:TRACE,p2p:TRACE,debug:DEBUG`. Stop one validator from shard X, replace its validatorKey.pem with another validator from shard X and then start it. We should be able to check in the logs that all nodes from shard X received at least 2 messages from that public key
